trilitech/jstz
# Node: persistent logging feature

# Context

The presistent logging gives warning in the sadbox environement due to the deterministc nature of the request id. we want to be able to disable the persistnet logging
[Task](https://app.asana.com/0/1205763124050213/1206544923875669) 

# Description

Add a feature to be able to turn on the persistent log.

# Manually testing the PR

run the sandbox and do 
`sqlite3 ~/.jstz/log.db "SELECT * FROM log"`
to see if the flushing is diabled after running the contract
